storage/sql_migrations/004_statuslist.sql

```diff
@@ -0,0 +1,43 @@
+-- migrate:up
+
+-- status_list_credentials: keeps track of all status list credentials issued by an issuer, and the highest status list index issued for each credential.
+create table status_list_credentials
```

Comment:
I used plural because its the convention used in GORM. I don't really care either way, but singular would create column `status_list_credential.status_list_credential` if that is fine?

Reply:
I'd name the column `url`?

Reply:
I was naming from the statusListEntry point of view, but viewing from the credential makes much more sense